modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -16,16 +16,18 @@ class PowerOfAttorneyController < ClaimsApi::BaseFormController
         FORM_NUMBER = '2122'
 
         def submit_form_2122
-          power_of_attorney = ClaimsApi::PowerOfAttorney.create(
-            status: ClaimsApi::PowerOfAttorney::PENDING,
-            auth_headers: auth_headers,
-            form_data: form_attributes,
-            source_data: {
-              name: source_name,
-              icn: current_user.icn,
-              email: current_user.email
-            }
-          )
+          power_of_attorney = ClaimsApi::PowerOfAttorney.find_by(header_md5: header_md5)
+          unless %w[submitted pending].include? power_of_attorney.try(:status)
+            power_of_attorney = ClaimsApi::PowerOfAttorney.create(
+              status: ClaimsApi::PowerOfAttorney::PENDING,
+              auth_headers: auth_headers,
+              form_data: form_attributes,
+              source_data: source_data,
+              current_poa: current_poa,
+              header_md5: header_md5
+            )
+          end
+
           power_of_attorney = ClaimsApi::PowerOfAttorney.find_by(md5: power_of_attorney.md5) unless power_of_attorney.id
```

Comment:
same as my comment above

Reply:
I can bring this into the loop if you prefer

Reply:
Moved it into the loop